docs/contributing/rdg-storage-format.rst

```diff
@@ -0,0 +1,61 @@
+=========================
+Katana RDG Storage Format
```

Comment:
I'm surprised that we didn't explain what an RDG was in any other portion of the contributing docs.